### ISSUE TITLE
crash visiting rockney

### ISSUE THREAD

**Issue by thatmacguy** [michaelchadwick/gemwarrior#56]
```
 GW> u rockney
You're not sure what to expect when you confront the small animal living in the
crevice, but you figure it's this or doing anything else at all, so...

undefined local variable or method `player' for #<Gemwarrior::Rockney:0x007fd94d16df40>
######################################################################
Gem Warrior v0.11.3 played for 5875 min(s), 13 sec(s), and 913 ms
```

**Comment by thatmacguy** [comment]
exists in v0.11.4 too

```
 GW> u rockney
You're not sure what to expect when you confront the small animal living in the
crevice, but you figure it's this or doing anything else at all, so...

undefined local variable or method `player' for #<Gemwarrior::Rockney:0x007fd05a2c60f0>
######################################################################
Gem Warrior v0.11.4 played for 5875 min(s), 23 sec(s), and 94 ms
```

**Comment by michaelchadwick** [comment]
This has been fixed in [0.11.5](https://github.com/michaelchadwick/gemwarrior/releases/tag/v0.11.5).
